### PR TITLE
Add blog post on "Meteor and FerretDB"

### PR DESCRIPTION
**Pull request by Fashander**
# Description

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/blog/2023-05-17-ferretdb-meteor-mongodb-alternative.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+The FerretDB team is actively addressing these limitations.
+To stay up to date with the compatibility status and ongoing efforts, please refer to the [FerretDB documentation](https://docs.ferretdb.io/reference/supported-commands/), or take a look at this [issue where FerretDB aims to achieve compatibility with Meteor.js](https://github.com/FerretDB/FerretDB/issues/2414) examples.
+
+While dealing with these compatibility issues, one of the issues the FerretDB team has grappled with is the challenge of supporting `OpLog` tailing.
+The introduction of  `ChangeStream` in MongoDB 3.6 and later allows real-time data access that bypasses the complications associated with `OpLog` tailing.
+
+Even though Meteor.js traditionally supports MongoDB’s `OpLog` tailing for real-time data sync, Meteor.js falls back to polling when `OpLog` is not available; however, while this is not ideal for many production use cases, it can occasionally do the job.
+At this stage, it’s unclear whether the hassles of implementing `OpLog` will be worth it, especially with a few Meteor.js apps, like Rocket.Chat, moving to `ChangeStream`.
```
[the remark this answers is not in the document]
Comment (Fashander) [reply website/blog/2023-05-17-ferretdb-meteor-mongodb-alternative.md]: @AlekSi Can you please take a look at this section? Everything else has been approved by Peter.

File: website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Since MongoDB's license switch to SSPL from its former open-source license, there have been growing concerns about the nature of the license and how it may affect developers and companies building their applications with MongoDB.
+Several Meteor.js application developers are also in this corner.
+
+And this is what makes [the announcement of the General Availability version](https://blog.ferretdb.io/ferretdb-1-0-ga-opensource-mongodb-alternative/) of FerretDB – an open-source replacement to MongoDB – so exciting!
+
+## Introducing FerretDB
+
+FerretDB is the defacto open-source MongoDB alternative that converts MongoDB wire protocols with a backend on [PostgreSQL](https://www.postgresql.org/).
+Besides PostgreSQL, FerretDB also supports other database backends, such as [Tigris](https://www.tigrisdata.com/), [SAP Hana](https://www.sap.com/africa/products/technology-platform/hana.html), [SQLite](https://sqlite.org/), and many more to come.
```
Comment (AlekSi) [inline website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md:46]: https://docs.ferretdb.io/understanding-ferretdb/#supported-backends is pretty clear that SQLite and Hana are not supported yet. They don't even work

**Review by ptrfarkas — DISMISSED** [review]

File: website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+* **User-friendliness:** FerretDB lets you jump in and use the same syntax you're familiar with, so there's no need to learn a new language or syntax; it's the same familiar environment you're already used to.
+* **PostgreSQL backend:** FerretDB's database engine, PostgreSQL, is renowned for its open-source nature, reliability, and robust community, as well as its ability to handle large amounts of data and complex queries, which can be an added advantage.
+
+## Replacing MongoDB With FerretDB in Your Meteor.js Apps
+
+FerretDB has made significant strides in ensuring compatibility with Meteor.js apps, and the development team continues to work on enhancing this compatibility further.
+As of the latest release, FerretDB demonstrates compatibility with many common MongoDB use cases encountered in Meteor.js applications.
+
+The current level of compatibility enables Meteor.js developers to seamlessly transition their applications from MongoDB to FerretDB with minimal friction.
+The current level of compatibility enables Meteor.js developers, already accustomed to MongoDB, to seamlessly transition to FerretDB with minimal friction.
```
Comment (AlekSi) [inline website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md:68]: That's the same sentence

File: website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+The FerretDB team is actively addressing these limitations.
+To stay up to date with the compatibility status and ongoing efforts, please refer to the [FerretDB documentation](https://docs.ferretdb.io/reference/supported-commands/), or take a look at this [issue where FerretDB aims to achieve compatibility with Meteor.js](https://github.com/FerretDB/FerretDB/issues/2414) examples.
+
+While dealing with these compatibility issues, one of the issues the FerretDB team has grappled with is the challenge of supporting `OpLog` tailing.
+The introduction of  `ChangeStream` in MongoDB 3.6 and later allows real-time data access that bypasses the complications associated with `OpLog` tailing.
+
+Even though Meteor.js traditionally supports MongoDB’s `OpLog` tailing for real-time data sync, Meteor.js falls back to polling when `OpLog` is not available; however, while this is not ideal for many production use cases, it can occasionally do the job.
+At this stage, it’s unclear whether the hassles of implementing `OpLog` will be worth it, especially with a few Meteor.js apps, like Rocket.Chat, moving to `ChangeStream`.
```
Comment (AlekSi) [inline website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md:79]: https://xkcd.com/285/

File: website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+The same query language and syntax can be directly applied to FerretDB, and this compatibility also extends to MongoDB management tools like Compass, Mingo UI, Studio 3T, etc.
+
+However, please note that while FerretDB strives to achieve comprehensive compatibility with Meteor.js apps, there may be certain edge cases or specific MongoDB features that are not yet fully supported.
+The FerretDB team is actively addressing these limitations.
+To stay up to date with the compatibility status and ongoing efforts, please refer to the [FerretDB documentation](https://docs.ferretdb.io/reference/supported-commands/), or take a look at this [issue where FerretDB aims to achieve compatibility with Meteor.js](https://github.com/FerretDB/FerretDB/issues/2414) examples.
+
+While dealing with these compatibility issues, one of the issues the FerretDB team has grappled with is the challenge of supporting `OpLog` tailing.
+The introduction of  `ChangeStream` in MongoDB 3.6 and later allows real-time data access that bypasses the complications associated with `OpLog` tailing.
```
Comment (AlekSi) [inline website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md:76]: They are `ChangeStreams`, with an `s`

File: website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md [hunk 6]
```diff
@@ -64,19 +64,18 @@ Some of the benefits of using FerretDB with your Meteor.js applications include
...
-The introduction of  `ChangeStream` in MongoDB 3.6 and later allows real-time data access that bypasses the complications associated with `OpLog` tailing.
+While dealing with these compatibility issues, one of the [issues the FerretDB team has grappled with is the challenge of supporting `OpLog` tailing](https://github.com/meteor/meteor/discussions/12150).
+The introduction of  `ChangeStreams` in MongoDB 3.6 and later allows real-time data access that bypasses the complications associated with `OpLog` tailing.
 
 Even though Meteor.js traditionally supports MongoDB’s `OpLog` tailing for real-time data sync, Meteor.js falls back to polling when `OpLog` is not available; however, while this is not ideal for many production use cases, it can occasionally do the job.
-At this stage, it’s unclear whether the hassles of implementing `OpLog` will be worth it, especially with a few Meteor.js apps, like Rocket.Chat, moving to `ChangeStream`.
+At this stage, it’s unclear whether the hassles of implementing `OpLog` will be worth it, especially with a few Meteor.js apps, like Rocket.Chat, moving to `ChangeStreams`.
```
Comment (AlekSi) [inline website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md:78]: > especially with a few Meteor.js apps, like Rocket.Chat, moving to `ChangeStreams`.

[citation needed]

**Pull request merged**